.github/workflows/sql-test-and-build-workflow.yml

```diff
@@ -3,7 +3,7 @@ name: SQL Java CI
 on: [push, pull_request]
 
 env:
-  OPENSEARCH_VERSION: '2.0.0-SNAPSHOT'
```

Comment:
this should stay as 2.0.0-SNAPSHOT, then add another variable for `alpha1`, and update steps?
```yaml
    - name: Build with Gradle
      run: ./gradlew build assemble -Dopensearch.version=${{ env.OPENSEARCH_VERSION }} -Dbuild.version_qualifier=${{ env.QUALIFIER }}
```

Reply:
ignore this comment, although i saw they still want us to include `-Dbuild.version_qualifier=`